css/index.css

```diff
@@ -104,14 +143,89 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+/*Sprint Challenge: change colors of boxes (.boxes nth-child)*/
+
```

Comment:
Great job using :nth-child() selector to target the boxes, you don't have to change anything on the boxes in the HTML file this way.  You also make use of parent/child relationships and let the parent provide most the styling for each individual box.